tests/middleware_exceptions/tests.py

```diff
@@ -486,6 +486,15 @@ def test_process_template_response_error(self):
         # Check that the right middleware methods have been invoked
         self.assert_middleware_usage(middleware, True, True, True, True, False)
 
+    @override_settings(ROOT_URLCONF='middleware_exceptions.urls',
```

Comment:
preferred style:
```
@override_settings(
   ROOT_URLCONF=...
)
```
although ROOT_URLCONF is already specified on the class level -- does it need to be repeated?